js/signin.js

```diff
@@ -41,6 +41,21 @@ function eyeBlink(input,icon){
         icon.src="../images/eye-off.svg"
     }
 }
+
+//엑세스 토큰 저장
+function containAccessToken(accessToken) {
+    localStorage.setItem('accessToken', accessToken);
+}
+
+//엑세스 토큰 저장 여부 확인
+function haveAccessToken() {
+    const accessToken = localStorage.getItem('accessToken');
+    if (accessToken) {
+        // accessToken이 있는 경우에는 '/folder' 페이지로 이동
+        window.location.href = '/folder.html';
+    }
+}
```

Comment:
그래서 만약 여기를 util로 분리하게 된다면, 인자로 callback을 받고, if문 안에서 성공한 경우에 callback();이렇게 호출하시면 될 것 같습니다.

```
```suggestion
function haveAccessToken(callback) {
    const accessToken = localStorage.getItem('accessToken');
    if (accessToken) {
        // accessToken이 있는 경우에는 '/folder' 페이지로 이동
        callback();
    }
}
```

`haveAccessToken(()=>{window.location.href = '/folder.html';})` 이런식으로요!